CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
+  - Support for non-admin users to EMS Refresh
+  - Tenant relationships added to summary screens 
+- Amazon
+  - AWS Region EU Frankfurt
+  - Inventory collection for AWS CloudFormation
+  - Parsing of parameters from orchestration templates
+  - Providers: Amazon Events
+  - Enables event-based policies for AWS
+  - Added a backend attribute to identify public images.
+  - Added C4, D2, and G2 instance types.
+  - Virtualization type collected during EMS refresh for better
+    filtering of available types during provisioning.
+- Orchestration
  - Orchestration Stacks includes tagging
  - Orchestration templates
      - Create, edit, delete, tagging, ‘draft’ support
      - Can create a Service Dialog from template contents
- - HTML5 Consoles for OpenStack
-
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Aenhancement) 
-
-- Automate
- - Dynamic Dialog Date/Time Control
- - Disabled domains clearly marked in UI
- - Automate entry point selection reduced to state machine classes
-- IPv6
- - More code paths communicate via hostname
- - URI building with IPv6 safety
-- UI: tagging support and toolbars on list views
-
-### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Abug)
-
-- Ruby2 trap logging and worker row status
-- Update STMP domain length to match SMTP host length
-
-## Unreleased - as of Sprint 20 end 2015-03-09
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+20+Ending+Mar+9%2C+2015%22+label%3Aenhancement)
+ - Provisioning dialog generator
+ - Enabled Reporting / Tagging
+     
+#### Provisioning
+- Heat Orchestration provisioning through services
+- Foreman
+  - Provisioning of bare metal systems
+  - Organization and location inventory
+  - Uses latest Foreman Apipie gem
+- Allow removing keys from :clone_options by setting value to nil
+- OpenStack: Added tenant filtering on security groups, floating IPs, and
+    networks.
+- Amazon: Filtering of flavors based on root device type and block
+    storage restrictions.
 
-- UI
-  - Implement HTML5 Console for RHEVM and VMware
-  - Menu plugins for external sites
-  - Orchestration templates UI updates
-  - HAML and I18n strings 100% completed in views
-- Automate
-  - New retirement workflow
+#### User Interface
+- HTML5 Console for RHEVM, VMware, and OpenStack
+- Menu plugins for external sites
+- HAML and I18n strings 100% completed in views
+- Multi-character set language support
+- Can now set the locale for both server and user
+- Repository Editor using AngularJS
+- Charting updates: jqPlot, default charts, chart styling, donut chart support
+- Updates to form buttons with Patternfly
+- Tagging support and toolbars on list views
+- Login screen converted to Bootstrap / Patternfly
+- Header, navigation, and outer layouts converted to Bootstrap / Patternfly
+- Advanced search converted to Bootstrap / Patternfly
```

Comment:
Might want a subsection for Bootstrap / Patternfly changes and then bucket all these things under it.